karma.mc.config.js

```diff
@@ -26,6 +26,7 @@ module.exports = function(config) {
     files: [PATHS.testEntryFile],
     preprocessors,
     webpack: {
+      devtool: "inline-eval",
```

Comment:
The webpack 2.0 docs don't mention inline-eval.  Is this the same as just "eval"?  If so, I'd suggest switching to the canonical name.  If not, a comment would be helpful.